plutus-core/plutus-core/src/PlutusCore/Parsable.hs

```diff
@@ -112,6 +114,8 @@ instance Parsable (a, b) where
 instance Parsable ByteString where
     parse = parseByteStringConstant
 
+instance Parsable Data where
+    parse = error "Implement me"
```

Comment:
@michaelpj let's let @thealmarty handle that?

Reply:
yeah, happy to worry about parsing later